shared/desktop/app/menu-bar.js

```diff
@@ -93,19 +94,34 @@ export default function(menubarWindowIDCallback: (id: number) => void) {
       if (isWindows) {
         const cursorPoint = electronScreen.getCursorScreenPoint()
         const screenSize = electronScreen.getDisplayNearestPoint(cursorPoint).workArea
-        if (screenSize.x > 0) {
-          // start menu on left
-          mb.setOption('windowPosition', 'trayBottomLeft')
-        } else if (screenSize.y > 0) {
-          // start menu on top
-          mb.setOption('windowPosition', 'trayRight')
-        } else if (cursorPoint.x > screenSize.x) {
-          // start menu on right
-          mb.setOption('windowPosition', 'bottomRight')
+        let menuBounds = mb.window.getBounds()
```

Comment:
lets be extra careful here and check mb.window and mb.tray (since the old code only ever called setOption)